inventory.d/signing.yaml

```diff
@@ -30,3 +30,15 @@ groups:
       - dep-mac-v3-signing3.srv.releng.mdc1.mozilla.com
     facts:
       puppet_role: mac_v3_signing_dep
+  - name: mac-v3-signing-mozillavpn
+    targets:
+      - macmini-r8-180.test.releng.mdc1.mozilla.com
+      - macmini-r8-182.test.releng.mdc1.mozilla.com
+    facts:
+      puppet_role: mac_v3_signing_mozvpn
+  - name: mac-v3-signing-mozillavpn
```

Comment:
I think these two are just for building? If so we should probably drop `signing` from this name + the role (it took me an awful long time to figure out what why we had two `signing` entries...)

Reply:
sorry for the confusion. I was using the name for the puppet role and expecting they were using the "signing" scriptworker instead of another taskcluster worker.